gameobject.cpp

```diff
@@ -0,0 +1,17 @@
+#include "gameobject.h"
+
+#include "renderutils.h"
+
+void GameObject::Render()
+{
+    ASSERT(mMesh != nullptr);
+    mMesh->Draw(mPosition);
+}
+
+void GameObject::OnStart()
+{
+}
```

Comment:
I figured leaving these empty in the base class is the nicest. So that if a gameobject doesn't need to implement a OnUpdate() it doesn't have to override to implement a empty one.